packages/forms-web-app/src/lib/dashboard-functions.js

```diff
@@ -82,6 +87,15 @@ const formatAddress = (appealCaseData) => {
 	return addressComponents.filter(Boolean).join(', ');
 };
 
+const calculateAppealDueDeadline = (appealType, originalCaseDecisionDate) => {
+	if (appealType === 'HAS') {
+		return householderApplication(originalCaseDecisionDate);
+	} else if (appealType === 'S78') {
+		return fullAppealApplication(originalCaseDecisionDate);
+	}
+	return 'No due date';
+};
+
```

Comment:
No other code uses these, so I think we should avoid them (I'll make a note to delete them)!

Looks like we should use `businessRulesDeadline`, use `rawDate = true` for a `Date` object return value. For appeal type it uses the `APPEAL_ID` constants in business rules. we could add a mapping from `appealTypeCode` to `APPEAL_ID` in business rules and use that to pass to `businessRulesDeadline`.

Reply:
PR here to remove them: https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/3378